config/default/manager_default_images.yaml

```diff
@@ -19,7 +19,7 @@ spec:
           value: quay.io/podified-antelope-centos9/openstack-cron:current-podified
         - name: RELATED_IMAGE_OPENSTACK_EDPM_NOVA_COMPUTE_DEFAULT_IMG
           value: quay.io/podified-antelope-centos9/openstack-nova-compute:current-podified
-        - name: RELATED_IMAGE_OPENSTACK_EDPM_NOVA_LIBVIRT_DEFAULT_IMG
+        - name: RELATED_IMAGE_OPENSTACK_EDPM_LIBVIRT_DEFAULT_IMG
```

Comment:
If you change this ENV variable you will also need to adjust the name in SetupAnsibleImageDefaults() function within controllers/openstackdataplanenodeset_controller.go

Reply:
thanks. I missed that initially.